swan-lake/featured-use-cases/build-a-data-service-in-ballerina.md

```diff
@@ -259,7 +269,7 @@ import ballerina/http;
 
 ### Create a service
 Now, you can create a service as shown below. This creates an `/employees` endpoint on port `8080` which can 
-be accessed via a browser by visiting `http://locahost:8080/employees` after executing the command `bal run`.
+be accessed by visiting `http://localhost:8080/employees` after executing the command `bal run`.
```

Comment:
```suggestion
be accessed by visiting `http://localhost:8080/employees` after executing the `bal run`  command.
```

Reply:
Fixed in [82417aa](https://github.com/ballerina-platform/ballerina-dev-website/pull/5041/commits/82417aa7d92ccb948ab137f2b8ee79856a286b0c)